mod-source-record-manager-server/src/test/java/org/folio/verticle/consumers/RawMarcChunkConsumersVerticleTest.java

```diff
@@ -67,8 +63,6 @@ public static void setUpClass() throws IOException {
   public void setUp() {
     WireMock.stubFor(WireMock.get("/data-import-profiles/jobProfiles/" + JOB_PROFILE_ID + "?withRelations=false&")
       .willReturn(WireMock.ok().withBody(Json.encode(new JobProfile().withId(JOB_PROFILE_ID).withName("Create instance")))));
-    WireMock.stubFor(WireMock.post("/source-storage/batch/verified-records")
```

Comment:
@bisecomsergiy, please return back this stub. Thank you!

Reply:
Thank you Ruslan, it is done